.github/workflows/chart.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish_helm_chart
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
`release-v*` too?

edit: Err I guess its not until there's a tag that we really want to build the chart hm

Reply:
That's right! So we need to only publish from `main`/`master`.